src/ert/gui/simulation/view/progress_widget.py

```diff
@@ -2,22 +2,21 @@
 
 from typing import Any
 
-from qtpy.QtGui import QColor
-from qtpy.QtWidgets import (
+from PySide6.QtGui import QColor
+from PySide6.QtWidgets import (
     QFrame,
     QHBoxLayout,
     QLabel,
     QProgressBar,
     QVBoxLayout,
-    QWidget,
 )
 
 from ert.ensemble_evaluator.state import ENSEMBLE_STATE_FAILED, REAL_STATE_TO_COLOR
 
 
 class ProgressWidget(QFrame):
     def __init__(self) -> None:
-        QWidget.__init__(self)
+        QFrame.__init__(self)
```

Comment:
Should we change this to `super().__init__()?

Reply:
we should :)